opt/build/tflite/Dockerfile.x64

```diff
@@ -1,29 +1,34 @@
+ARG BAZEL_VERSION=3.1.0
+ARG TFLITE_ARCH=x86_64
 
-ARG OS=debian:buster
+ARG OS=redisfab/ubuntu1804-${TFLITE_ARCH}-bazel${BAZEL_VERSION}
```

Comment:
Which container is that? ubuntu:bionic with bazel of a particular version for the given architecture? Notice that Bazel versions vary frequently, and we might not match a requirement for a specific one.

Reply:
This is a container built, using the automata repo and pushed to redisfab. It specifically has the bazel version required for tflite as there's a hard requirement in the build (i.e 3.1.0).